test/k8s-integration/main.go

```diff
@@ -355,6 +391,32 @@ func generateGCETestSkip(testParams *testParameters) string {
 	return skipString
 }
```

Comment:
I would have expected kustomize changes somewhere to use the new overlays. Or am I missing something?

Reply:
From offline conversation: it's passed in via run-k8s-integration*.sh and which will appear in a separate PR.